src/documents/InfusionEventSystem.md

```diff
@@ -124,12 +135,63 @@ listeners: {
+Secondly, rather than a simple string, the key in a `listeners` structure can hold any [IoC Reference](IoCReferences.md) which resolves to an event anywhere in the component tree - 
+that is, even one belonging to a different component. In this case the listener on the right hand side will be attached to that event rather than one of this component's own events. 
+The framework will make sure to automatically deregister the listener when this component is destroyed. Many more complex cases are possible, including the wholesale injection of events from one component to another, 
+and the creation of new events derived from existing ones. You can consult the page [Event injection and boiling](EventInjectionAndBoiling.md) for more details.
+
+### Full listener record form
+
+In many cases, the short form above where the listener declaration just consists of a single `String` doesn't provide enough space for all the directives that you want to supply for how the listener is notified.
+In this case, you can use the long form record, where the listener record takes the form of a JSON structure with the following members (the key for this record stays the same as above):
+
+<table>
+  <thead>
+    <tr>
+      <td>Member Name</td>
+      <td>Type</td>
+      <td>Description</td>
+    </tr>
+  </thead>
+  <tr>
+    <td><code>listener</code>, <code>func/funcName</code></td>
+    <td><code>String</code></td>
+    <td>This holds the actual designation of the function which is to be the listener, which was the string that consisted of the entire "short form" for the listener definition above. This, as before, holds either 
+    the name of a global function or an IoC reference to a function. The `func`/`funcName` form is supported for consistency with the syntax for [Invokers](Invokers.md) and the name `listener` will be deprecated for the Infusion 2.0 release. 
+    </td>
+  </tr>
+  <tr>
+    <td><code>args</code> (optional)</td>
+    <td>Any JSON type</td>
+    <td>If supplied, this structure will be used to replace the actual arguments received by the listener, rather than necessarily receiving the exact signature fired by the firer. Any [IoC references](IoCReferences.md) or [expanders](ExpansionOfComponentOptions.md) will be expanded
+    at the time of invoking the listeners, with the additional possibility of referring to the original argument list by using the special context `{arguments}'. Consult [Event Injection and Boiling](EventInjectionAndBoiling.md) for more details of this process.</td>
```

Comment:
The close quote on {arguments} needs to be a back-quote. All the following text is rendering as code-styled.